cypress-tests/cypress/e2e/v2/verifyInfrastructure.cy.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Retry invoking a Cypress request until it succeeds or fail a few times in a row.
+ * @param url - target URL
+ * @param service - name of the tested service
+ * @param limit - maximum attempts before failing
+ * @param delaySeconds - delay between attempts
+ * @returns response body from the request or `null`
+ */
+function retryRequest(
+  url: string,
+  service: string,
+  limit = 6,
+  delaySeconds = 20,
+  retries = 1,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+): Cypress.Response<any> {
```

Comment:
You are not using the return value:

```suggestion
) {
```